src/wp-admin/includes/update-core.php

```diff
@@ -798,6 +798,8 @@
 	'wp-admin/css/ie.min.css',
 	'wp-admin/css/ie-rtl.css',
 	'wp-admin/css/ie-rtl.min.css',
+	// 5.6
+	'wp-includes/block-supports/index.php',
```

Comment:
This file only existed on trunk and was never released, do we need to add it here anyway?